include/swift/SIL/BranchPropagatedUser.h

```diff
@@ -96,6 +107,19 @@ class BranchPropagatedUser {
     NumLowBitsAvailable =
         llvm::PointerLikeTypeTraits<InnerTy>::NumLowBitsAvailable
   };
+
+private:
+  BranchPropagatedUser(SILInstruction *inst) : user(inst) {
```

Comment:
These are still used by some internal details of BranchPropagatedUser. I wanted to keep this commit really clean and easy to read, so I made these private rather than modifying the in-class callers.